arcgis-ios-sdk-samples/Search/Find place/FindPlaceViewController.swift

```diff
@@ -86,27 +84,24 @@ class FindPlaceViewController: UIViewController {
             }
         }
         
-        // add the graphicsOverlay to the map view
-        self.mapView.graphicsOverlays.add(graphicsOverlay)
-        
-        // initialize locator task
-        self.locatorTask = AGSLocatorTask(url: URL(string: "https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer")!)
+        // Initialize locator task.
+        self.locatorTask = AGSLocatorTask(url: URL(string: "https://geocode-api.arcgis.com/arcgis/rest/services/World/GeocodeServer")!)
```

Comment:
Could `locatorTask` be initialized with this value and made both non-optional and immutable?